src/App.js

```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+let DATA = [
+  {
+    id: 1,
+    title: "Lorem ipsum dolor sit amet consectetur",
+  },
+  {
+    id: 2,
+    title: "Lorem ipsum dolor sit amet consectetur",
+  },
+];
+function ListTodos({ title, id, deleteTodo }) {
+  const [isCheck, setIsCheck] = React.useState(false);
+  return (
+    <div className="flex justify-between items-center py-3 border-b">
+      <div className="flex gap-10 items-center">
+        <input type="checkbox" onClick={() => setIsCheck(!isCheck)} />
+        {isCheck ? <p className="line-through">{title}</p> : <p>{title}</p>}
+      </div>
+      <button
+        className="text-red-700 bg-slate-200 rounded-full w-10 h-10 flex justify-center items-center text-base"
+        onClick={() => deleteTodo(id)}
+      >
+        <FontAwesomeIcon icon={faTrash} />
+      </button>
+    </div>
+  );
+}
 
-function App() {
+export default function Todos() {
+  const [data, setData] = React.useState(DATA);
+  const [inputValue, setInputValue] = React.useState();
+  function addNewTodo(title) {
+    setData((data) => [
+      ...data,
+      { id: data[data.length - 1]?.id + 1 || 1, title: title },
+    ]);
+    setInputValue("");
+  }
+  function deleteTodo(id) {
+    let newValue = data.filter((item) => item.id !== id);
+    setData(newValue);
+  }
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <div className="flex justify-center items-center font-bold text-2xl">
+      <div className="w-[40%] text-center">
+        <h1 className="text-6xl">Todos</h1>
+        <form
+          className="flex items-center text-lg w-full  shadow-xl border p-3 rounded-full"
+          onSubmit={(e) => e.preventDefault()}
         >
-          Learn React
-        </a>
-      </header>
+          <input
+            type="text"
+            value={inputValue || ""}
```

Comment:
To reduce complexity, by default you can initialize the input value state with `""` at the top. And then remove it here